Dockerfile

```diff
@@ -1,24 +1,26 @@
-FROM openjdk:8-jre-slim
-LABEL maintainer="Dimagi <devops@dimagi.com>"
-
-# Environment variable WEB_HOST is used for setting where Django is
-# running; either in another container ("web") or on the Docker host
-# machine ("dockerhost").
-#
-# /formplayer/set_webhost adds the IP address of the host machine to
-# /etc/hosts and sets the hostname "webhost" as an alias to HQ.
+FROM adoptopenjdk:8-jre-hotspot as builder
+WORKDIR application
+ARG JAR_FILE=build/libs/*.jar
+COPY ${JAR_FILE} formplayer.jar
```

Comment:
What is the workflow for updating formplayer?

Reply:
I'm still working on that but it's basically this:
```
./gradlew bootJar
docker build -t dimagi/formplayer .
```

Reply:
Sweet!